js/main.js

```diff
@@ -229,11 +229,16 @@ $(window).bind('keydown', function(e) {
         // Select all text in address bar
         $('.js-navAddressBar').select();
         break;
+      case config.keyShortcuts.save:
+        // Trigger any visible .saveForm button
+        // .last() will make sure you'll only click one button if 2 are visible (listing form)
+        $('.js-saveForm:visible').last().trigger('click');
```

Comment:
Using the class and code in main.js means each view has code acting on it from outside the view, which can introduce bugs later. Using a listener means each view can determine for itself what happens when the shortcut is fired, which is more futureproof. 

The listener in most cases will be very simple, and makes the code easier to understand, as you only need to look at the view in question to understand all the actions in that view.

I know there's a universal link handler in main.js, we debated that quite a bit before adding it, since it's not a very good practice. In general having any code outside a view change the behavior of code in that view isn't ideal when we're using Backbone and Backbone patterns.